config/cloud-config/node-config-template.yaml

```diff
@@ -44,7 +47,7 @@ write_files:
       echo " " >> /etc/network-environment
```

Comment:
We should get rid of the whole /etc/network-environment thing for configuring Calico and use the config file method instead - it will simplify this a bunch.